perf/benchmark.cjs

```diff
@@ -54,6 +54,11 @@ const argv = yargs(hideBin(process.argv))
     default: false,
     description: 'Do not check for local changes, your local changes will be automatically stashed and un-stashed',
   })
+  .option('print-confidence', {
+    type: 'boolean',
+    default: false,
+    description: 'Print confidence range in reports',
```

Comment:
```suggestion
    description: 'Print 95 % confidence range in reports instead of simple +X% (increase "Number of sample" to reduce this range)',
```

Reply:
```suggestion
    description: 'Print 95 % confidence range in reports instead of +X% (increase "Number of sample" to reduce this range)',
```

Reply:
```suggestion
    description: 'Print 95 % confidence range in reports instead of +X% (increase the number of samples to reduce this range)',
```